<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>    登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix2/rc/loginicon.png</normaloff>:/new/prefix2/rc/loginicon.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>欢迎使用德语背单词软件，请您登录</string>
   </property>
   <widget class="QComboBox" name="usernameComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mimalabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rc.qrc">:/new/prefix2/rc/mimaicon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="remembercheckBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creatNewuserButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新建用户</string>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/new/prefix1/rc/actions/user_group_new.png</normaloff>:/new/prefix1/rc/actions/user_group_new.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="forgetPasswordButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/new/prefix1/rc/actions/kgpg_identity_kgpg.png</normaloff>:/new/prefix1/rc/actions/kgpg_identity_kgpg.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="autocheckBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>自动登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="userlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rc.qrc">:/new/prefix2/rc/loginuser.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/apply.png</normaloff>:/new/prefix1/rc/actions/apply.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix1/rc/actions/editdelete.png</normaloff>:/new/prefix1/rc/actions/editdelete.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
